### PR TITLE
added FindHostByCustomIdentifier

### PR DESCRIPTION
**Pull request by yseto**

### CODE REVIEW

File: hosts_test.go [hunk 1]
```diff
@@ -151,6 +151,51 @@ func TestFindHosts(t *testing.T) {
 
 }
 
+func TestFindHostByCustomIdentifier(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
+		if req.URL.Path != "/api/v0/hosts-by-custom-identifier/mydb001" {
```
Comment (Arthur1) [inline hosts_test.go:156]: `name` and `customIdentifier` are the same value. So, in this test code, it is ambiguous from which parameter the URL string is generated.

Comment (Arthur1) [reply hosts_test.go:156]: `customIdentifier` can be any string. It is preferable to use `customIdentifier` that requires percent-encoding as a test case.

**Review by Arthur1 — COMMENTED** [review]
The implementation looks fine to us. Please check our suggestions about test codes.

**Review by Arthur1 — APPROVED** [review]
LGTM!

**Comment by yseto** [comment]
Thanks for your review.

**Pull request merged**